spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -19,10 +19,28 @@
         model = define_model :example do
           def self.statuses; end
         end
-        message = "Expected #{model} to define :statuses as an enum"
-        assertion = -> {
+        message = "Expected #{model} to define :statuses as an enum and store the value in a column with an integer type"
+        assertion = -> do
           expect(model.new).to define_enum_for(:statuses)
-        }
+        end
```

Comment:
Not sure why Hound isn't catching this, but there should be new lines around blocks.